pal/audio/operation-queue.ts

```diff
@@ -33,6 +33,7 @@ export interface OperationInfo {
     invoking: boolean,
 }
 
+/** @mangle */
 export interface OperationQueueable {
```

Comment:
`OperationQueueable` is not exported, so it's safe to mangle it to reduce code size.